dashboard/app/controllers/projects_controller.rb

```diff
@@ -68,15 +65,19 @@ def load
     end
 
     create_new
+  rescue CanCan::AccessDenied
+    redirect_unauthorized
```

Comment:
I believe the 'automatic' CanCan redirect happens using [this handler that we set up in application_controller.rb](https://github.com/code-dot-org/code-dot-org/blob/8f4f56b057751dc21a61a82c190212a68866ba43/dashboard/app/controllers/application_controller.rb#L63).

```ruby
  rescue_from CanCan::AccessDenied do
    if !current_user && request.format == :html
      # we don't know who you are, you can try to sign in
      authenticate_user!
    else
      # we know who you are, you shouldn't be here
      head :forbidden
    end
  end
```

I'm creating my own handler(s) here because I want to preserve the behavior of `ProjectsController` which is different from the above in the 'else' case - we want to redirect to home instead of returning a 403.  Whether we should this behavior is a different discussion.

Then again, maybe that `head :forbidden` line does something different than I think it does?